### PR TITLE
Code standards

### PR DESCRIPTION
**Pull request by serundeputy**

### CODE REVIEW

**Comment by serundeputy** [comment]
This PR can automate code standards checks on i18n with each PR/commit via travis and backdrop/coder.

Only travisCI is using composer not the users of `i18n`.

File: i18n.module [hunk 1]
```diff
@@ -260,7 +260,7 @@ function i18n_langcode($langcode = NULL) {
 /**
  * Implements hook_help().
```
Comment (indigoxela) [inline i18n.module:261]: This needs evaluation. hook_help exists in Drupal 7, but got dropped in Backdrop. So currently this code isn't run at all.

File: i18n.module [hunk 2]
```diff
@@ -422,15 +422,15 @@ function i18n_object($type, $object) {
...
  * Get object wrapper by object key.
  *
- * @param $type
+ * @param string $type
  *   The object type to load e.g. node_type, menu, taxonomy_term.
- * @param $key
+ * @param string $key
  *   The object key, can be an scalar or an array.
- * @param $object
+ * @param object $object
```
Comment (indigoxela) [inline i18n.module]: If beliefing the comment line below, this can be object or array. Needs evaluation.

File: i18n.module [hunk 3]
```diff
@@ -454,9 +454,9 @@ function i18n_get_object($type, $key, $object = NULL) {
 /**
  * Get object language code.
  *
- * @param $object
+ * @param object $object
  *   Object or array having language field or plain language field
- * @param $default
+ * @param bool $default
```
Comment (indigoxela) [inline i18n.module:459]: I don't think bool is correct here.

File: i18n.module (hunk 1 above)
Comment (serundeputy) [reply i18n.module:261]: yea; I think we should remove this `hook_help`

**Comment by serundeputy** [comment]
@indigoxela I pushed up and additional commit removing the `hook_help` function and adding `@param object|array` for line 429.

File: i18n.module (hunk 1 above)
Comment (indigoxela) [reply i18n.module:261]: What we actually should do is to move the help text to a proper place (form). And we should do that also for all the submodules that currently still have an implementation of this hook. I'd prefer to leave this untouched for the moment.

Comment (serundeputy) [reply i18n.module:261]: got it.

Comment (serundeputy) [reply i18n.module:261]: I put it back.

**Pull request merged**